src/SignhostAPIClient/Rest/SignHostApiClient.cs

```diff
@@ -15,8 +15,7 @@ namespace Signhost.APIClient.Rest
 	/// an signhost api client implementation.
 	/// </summary>
 	public class SignHostApiClient
-		: ISignHostApiClient
-		, IDisposable
+		: ISignHostApiClient, IDisposable
```

Comment:
Doesn't match our code-guidelines, we require every inheritance to be on its own line and it looks like the original is actually our preferred way of doing things.

Reply:
Stylecop code analyzer rules and code guidelines differ from one another.
Dropped this commit.